resources/views/forum/forums/index.blade.php

```diff
@@ -19,7 +19,7 @@
     'legacyNav' => false,
     'pageDescription' => trans('forum.title'),
     'searchParams' => ['mode' => 'forum_post'],
-    'useTorusFont' => true,
+    'use2019Font' => true,
```

Comment:
probably worth inverting this and having it be `useLegacyFont` or something instead?